internal/controller/githubissue_controller.go

```
@@ -224,15 +213,96 @@ func (r *GithubIssueReconciler) Reconcile(ctx context.Context, req ctrl.Request)
+		fmt.Print(item)
+		fmt.Printf("\nIssue %d:\n", i)
+		number, ok := item["number"].(string)
+		if ok {
+			numberList = []string{number}
+		} else {
+			return false, fmt.Errorf("can't retrieve issue number")
+		}
+		title := item["title"].(string)
+		description := item["body"].(string)
+		if !slices.Equal(listIssueNumber, numberList) {
+			r.createCR(title, description, repo, number, accessToken)
```

golangci-lint (1.64.8)

Error return value of `r.createCR` is not checked

(errcheck)